velox/docs/develop/operators.rst

```diff
@@ -619,6 +619,8 @@ If no sorting columns are specified then the order of the results is unspecified
     - Output column names for each window function invocation in windowFunctions list below.
   * - windowFunctions
     - Window function calls with the frame clause. e.g row_number(), first_value(name) between range 10 preceding and current row. The default frame is between range unbounded preceding and current row.
+  * - inputsSorted
+    - If the input data is already sorted, the StreamingWindow will be used to reduce the memory footprint.
```

Comment:
Nit : Maybe not use "StreamingWindow" here as it is not a thing anymore. Please describe the optimizations in the Window operator that result from this field.

Reply:
Done.